workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/request/client/HttpClient.java

```diff
@@ -29,13 +34,28 @@ public Response execute(String method, String url, Object body, Map<String, Stri
     int responseCode = httpResponse.getCode();
 
     if (classicHttpResponse.getEntity() != null && classicHttpResponse.getEntity().getContent() != null) {
-      return new Response(responseCode,
-          IOUtils.toString(classicHttpResponse.getEntity().getContent(), StandardCharsets.UTF_8));
+      return this.handleResponse(responseCode,
+          IOUtils.toString(classicHttpResponse.getEntity().getContent(), StandardCharsets.UTF_8),
+          classicHttpResponse.getFirstHeader(HttpHeaders.CONTENT_TYPE));
```

Comment:
should not we pass the value directly (to avoid checking the header's name again in handleResponse)?